tsconfig.json

```diff
@@ -5,14 +5,14 @@
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",
     "forceConsistentCasingInFileNames": true,
+    "esModuleInterop": true,
```

Comment:
Remind me tomorrow to check tsconfig once again

Reply:
also try to use this here: https://blog.angular.dev/using-isolatedmodules-in-angular-18-2-68a7d3a6c03d